sdk/cosmos/azure-cosmos/docs/TimeoutAndRetriesConfig.md

```diff
@@ -0,0 +1,31 @@
+## Cosmos DB Python SDK – Timeout configurations and Retry configurations
+
+### Timeout config
+
+The timeout options for the client can be changed from the default configurations with the options below. These are:
+- `Client Timeout`: can be changed by passing the `timeout` option. This option has no default value, and is only present
+if it is passed in by the user. If not present, the other connectivity timeouts will be used.
+- `Request Timeout`: can be changed by passing the `request_timeout` option. Default value is 60s.
+- `Connection Timeout`: can be changed through `connection_timeout` option. Default value is 60s.
+- `Read Timeout`: can be changed through `read_timeout` option. Default value is 300s.
+
+
+### Resource Throttle Retry Policy config
+
+The options for the resource throttle retry policy used for 429 error codes can be changed from the default client configurations with the options below. These are:
+- `Retry Total`: can be changed by passing the `retry_total` option. Default value is 9.
```

Comment:
9 seems odd... is that what we're using in Core?

Reply:
nope this is our own (Cosmos') default option being used for this particular policy:
<img width="649" alt="image" src="https://user-images.githubusercontent.com/30335873/236845441-4f7d1a32-7e0c-487f-b057-43c78325694a.png">

<img width="567" alt="image" src="https://user-images.githubusercontent.com/30335873/236845348-49582328-4773-4908-b801-5b174273462d.png">